easybuild/easyconfigs/p/PhyloGibbs/PhyloGibbs-1.2-goolf-1.4.10.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
```

Comment:
minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!